litmus-portal/frontend/src/pages/WorkflowInfoStats/WorkflowRunTable/TableToolbar.tsx

```diff
@@ -173,7 +175,7 @@ const TableToolBar: React.FC<TableToolBarProps> = ({
         variant="outlined"
         className={`${classes.formControl} ${classes.testForm}`}
       >
-        <InputLabel className={classes.selectText}> Test Name </InputLabel>
+        <InputLabel className={classes.selectText}>Experiment Name</InputLabel>
```

Comment:
Please add translations before GA

Reply:
translations will go as a separate PR, this is just for UI bug fixes